src/web/package.json

```diff
@@ -55,6 +55,8 @@
     "test:coverage": "vitest --coverage && open ./coverage/lcov-report/index.html"
   },
   "dependencies": {
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
```

Comment:
emotion looks to offer pretty nice ergonomics, but since we're forced into using SASS by the design system, I'm not sure if the extra dependency is needed. Thoughts on what the benefits or tradeoff of using this is, vs just using SASS?

Reply:
Great question @danielnaab!
In my opinion, the benefits of using a `css-in-js` solution like emotion are:

## Benefits

1. **Local Scoping:** styles on the component won't be overridden by other global css classes.
2. **Encapsulation:** modify all component-related code in the same place, without having to worry about unexpectedly changing other parts of the application. This also helps with delete-ability, if I remove this component all styles come with it.
3. **Portability:** We can safely move these components around without having to worry about the css specificity gotchas. 

## Trade Offs

Emotion has a little bit of a learning curve and  styled components could make the code a little harder to read.

Reply:
refactored to use scoped `scss` files per component.